Documentation/PageTsconfig/TceMain.rst

```diff
@@ -361,6 +361,7 @@ preview
     the GET-parameter name.
 
     Finally :ts:`additionalGetParameters` allow you to add arbitrary GET-parameters and even override others.
+    If your target page shows by default a list you also have to set sth. like :ts:`tx_myext_pi1.action = show`.
```

Comment:
Can you use "something" instead of "sth."?

Reply:
Sure.